features/manage-position/YourPosition.tsx

```diff
@@ -115,49 +115,64 @@ const YourPosition = () => {
   ) {
     return (
       <Container>
-        <Typography variant="h5">Your Position</Typography>
-        <Status>
-          <Label>Collateral supplied: </Label>
-          {`${_collateral} ${_collSymbol}`}
-        </Status>
-        <Status>
-          <Label>Tokens outstanding: </Label>
-          {`${_tokens} ${_tokenSymbol}`}
-        </Status>
-        <Status>
-          <Label>
-            Estimated identifier price (
-            <Link href={pricefeedUrl} target="_blank" rel="noopener noreferrer">
-              Coinbase Pro
-            </Link>
-            ):{" "}
-          </Label>
-          {_latestPrice}
-        </Status>
-        <Status>
-          <Label>(CR) Collateralization ratio:</Label>
-          {` ${pricedCR} (${_collSymbol} / ${_tokenSymbol})`}
-        </Status>
-        <Status>
-          <Label>(GCR) Global collateralization ratio:</Label>
-          {` ${pricedGCR} (${_collSymbol} / ${_tokenSymbol})`}
-        </Status>
-        <Status>
-          <Label>Liquidation Requirement:</Label>
-          {` ${collReqFromWei}`}
-        </Status>
-        <Status>
-          <Label>Liquidation Price:</Label>
-          {` ${liquidationPrice} (${priceIdUtf8})`}
-        </Status>
-        <Status>
-          <Label>Collateral pending/available to withdraw:</Label>
-          {` ${_withdrawAmt} ${_collSymbol}`}
-        </Status>
-        <Status>
-          <Label>Pending withdrawal request:</Label>
-          {` ${_pendingWithdraw}`}
-        </Status>
+        <Grid container spacing={4}>
+          <Grid item md={6} xs={12}>
+            <Typography variant="h5">Your Position</Typography>
+            <Status>
+              <Label>Collateral supplied: </Label>
```

Comment:
These <Labels> have a space at the end and no space preceding the string that follows.

I recommend changing all of the Labels to have a trailing space and remove the preceding space in the following text. So for example, change:

```
<Label>Collateral ratio (CR):</Label>
{` ${pricedCR}`}
```

to 

```
<Label>Collateral ratio (CR): </Label>
{`${pricedCR}`}
```

Reply:
Good catch. looks like this now:
![image](https://user-images.githubusercontent.com/12886084/88572308-3c375e00-d03f-11ea-9da4-2b3a96049520.png)